client/src/partials/journal/modals/columnsConfig.modal.html

```diff
@@ -0,0 +1,51 @@
+<form name="columnsConfigForm" ng-submit="ColumnsConfigModalCtrl.submit()" novalidate>
+
+  <div class="modal-header">
+    <h4>{{ "POSTING_JOURNAL.COLUMNS_CONFIG.TITLE" | translate }}</h4>
+  </div>
+
+  <div class="modal-body">
+
+    <p class="text-info">      
+      {{ "POSTING_JOURNAL.COLUMNS_CONFIG.HELP" | translate }}
+    </p>   
+
+    <hr/>
+
+    <div class="checkbox" ng-repeat="col in ColumnsConfigModalCtrl.columns">
+      <label>
+        <input
+          type="checkbox"
+          name="is_visible"
+          ng-true-value="true"
```

Comment:
Do you really need `ngTrueValue` and `ngFalseValue`?   Aren't these `true` and `false` by default?